.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
Please drop this (and the added/deleted files) from the history. `.idea` belongs in the user's global gitignore.